workflow/controller/operator.go

```diff
@@ -1667,3 +1674,23 @@ func (woc *wfOperationCtx) checkAndCompress() error {
 
 	return nil
 }
+
+func (woc *wfOperationCtx) SubstituteParamsInVolumes(params map[string]string) error {
```

Comment:
Can we make this method unexported seeing as it is not used outside.

Reply:
Done!